apps/sms/test/marionette/lib/views/composer/accessors.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+/* global module */
+
+var SELECTORS = Object.freeze({
+  main: '#thread-messages',
+  toField: '#messages-to-field',
+  recipientsInput: '#messages-to-field [contenteditable=true]',
+  recipients: '#messages-recipients-list .recipient[contenteditable=false]',
+  messageInput: '#messages-input',
+  subjectInput: '.subject-composer-input',
+  sendButton: '#messages-send-button',
+  attachButton: '#messages-attach-button',
+  header: '#messages-header',
+  charCounter: '.message-counter',
+  moreHeaderButton: '#messages-options-button',
+  mmsLabel: '.mms-label',
+  attachment: '#messages-input .attachment-container',
+  messageConvertNotice: '#messages-convert-notice',
+  recipientsList: '#messages-recipients-list'
+});
+
+function ComposerAccessor(client) {
+  this.client = client;
+}
+
+ComposerAccessor.prototype = {
+  get toField() {
+    return this.client.helper.waitForElement(SELECTORS.toField);
+  },
+
+  get recipientsInput() {
+    return this.client.helper.waitForElement(SELECTORS.recipientsInput);
+  },
+
+  get recipientsList() {
+    return this.client.helper.waitForElement(SELECTORS.recipientsList);
+  },
+
+  get recipients() {
+    return this.client.findElements(SELECTORS.recipients);
+  },
+
+  get messageInput() {
+    return this.client.helper.waitForElement(
+      SELECTORS.messageInput
+    );
+  },
+
+  get subjectInput() {
+    return this.client.helper.waitForElement(
+      SELECTORS.subjectInput
+    );
+  },
+
+  get sendButton() {
+    return this.client.helper.waitForElement(SELECTORS.sendButton);
+  },
+
+  get attachButton() {
+    return this.client.helper.waitForElement(
+      SELECTORS.attachButton
+    );
+  },
+
+  get header() {
+    return this.client.helper.waitForElement(SELECTORS.header);
+  },
+
+  get charCounter() {
+    return this.client.findElement(SELECTORS.charCounter);
+  },
+
+  get mmsLabel() {
+    return this.client.findElement(SELECTORS.mmsLabel);
+  },
+
+  get attachment() {
+    return this.client.findElement(SELECTORS.attachment);
+  },
+
+  get conversionBanner() {
+    return this.client.findElement(SELECTORS.messageConvertNotice);
+  },
+
+  get firstRecipient() {
+    return this.recipients[0];
+  },
+
+  waitToAppear: function() {
+    return this.client.helper.waitForElement(SELECTORS.main);
+  },
+
+  showOptions: function() {
```

Comment:
```showOptions``` feels like view method instead as well. If you think the same, could you please at least add TODO to move this to the view file?